CONTRIBUTING.md

```diff
@@ -187,6 +187,30 @@ All pull requests should be reviewed by:
 If there are any CODEOWNERs automatically assigned, you should also wait for
 their review.
 
+#### Bors review process
+
+Once you’ve reviewed the PR, instead of clicking the green “Merge Button”, leave a comment like this on the pull request:
+
+```text
+bors r+
+```
+
+Equivalently, you can comment the following:
+
+```text
+bors merge
+```
+
+The pull request, as well as any other pull requests that are reviewed around the same time, will be merged into a branch called `staging`. CI will run there and report the result back. If that result is “OK”, `master` gets fast-forwarded to reach it.
+
+There’s also:
+
+```text
+bors try
+```
+
+When this is run, your branch and master get merged into `trying`, and bors will report the results just like the `staging` branch would. Only reviewers can push to this.
+
```

Comment:
There's also `bors delegate+` (or `bors d+`), which allows reviewers to delegate `bors r+` to the PR author.

This allows us to review, ask for some minor changes, and tell the author that they can ask Bors to merge themselves, once they've addressed the feedback.

You wouldn't do this for PRs that require significant changes or more discussions, but for styling-only change requests, it will free you of having to circle back to the PR to `bors r+`, and gives more authority to PR authors when we think they can use that authority wisely.

Reply:
Yeah - I skipped that initially in the docs as a larger discussion on contributor workflow.